projects/core/src/site-theme/config/site-theme-config.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Config } from '../../config/config-tokens';
+import { SiteTheme } from '../../model/misc.model';
+
+@Injectable({
+  providedIn: 'root',
+  useExisting: Config,
+})
+export abstract class SiteThemeConfig {
+  siteTheme?: {
+    optionalThemes?: Array<SiteTheme>;
+  };
+}
```

Comment:
[Official docs reminder]
Let's make sure we document in our official docs for customers clearly the Site Theming configuration, e.g.:
- you can configure the default site theme value (css class) either via the Spartacus static config `config.context.theme` or you can auto-populate it from CMS, if you're using the feature of "automatic site-context configuration"
- you can configure the list of the optional themes (to be displayed in the `ThemeSwitcherComponent`) via the config `config.siteTheme.optionalThemes`